modules/client-configuration/pages/clients-sleses.adoc

```diff
@@ -74,6 +74,7 @@ The products you need for this procedure are:
 
 | {es} 8
 | RHEL or SLES ES or CentOS 8 Base and SUSE Linux Enterprise Server with Expanded Support 8 x86_64
+
```

Comment:
@keichwa I would say we need to add version 9 here too.

Reply:
ok, sounds sensible!